tests/claim.zil.test.ts

```diff
@@ -305,6 +332,32 @@ describe("staking contract", () => {
             await increaseBNum(zilliqa, 10);
           },
           error: undefined,
+          want: {
+            events: [
+              {
+                name: "check_rewards",
+                getParams: () => ({
+                  rewards: ["List (Pair (ByStr20) (Uint128))",
+                  [
+                    [globalToken1ContractAddress,"10000000000000"],
+                    [globalToken2ContractAddress,"10000000000000"],
+                  ]
+                ]
+                }),
+              }
```

Comment:
There are 4 events e.g. `setup_rewards_distribution`.
Can we add the events as well?

Reply:
Actually that one for debug/test only, about to remove this event from contract : )

Reply:
ok sure

Reply:
removed